src/model/resources/thermal_storage/thermal_storage.jl

```diff
@@ -165,13 +168,13 @@ function thermal_storage(EP::Model, inputs::Dict, setup::Dict)
 	)
 
 	# then for resources with RH
-	@constraint(EP, cTSocBalInterior[t in INTERIOR_SUBPERIODS, y in intersect(TS, RH)], (
+	@constraint(EP, cTSocBalInteriorRH[t in INTERIOR_SUBPERIODS, y in intersect(TS, RH)], (
 		vTS[y,t] == vTS[y,t-1]
 		- (1 / dfGen[y, :Eff_Down] * EP[:vP][y,t])
 		- (1 / dfGen[y, :Eff_Down] * dfGen[y, :Start_Fuel_MMBTU_per_MW] * dfGen[y,:Cap_Size] * EP[:vSTART][y,t])
 		+ (dfGen[y,:Eff_Up] * vCP[y,t])
-		- (dfGen[y,:Self_Disch] * vTS[y,t-1]))
-		+ (vRH[y, t])  #100% resistive heating efficiency
+		- (dfGen[y,:Self_Disch] * vTS[y,t-1])
+		+ (vRH[y, t]))  #100% resistive heating efficiency
```

Comment:
No need for `(` ... `)` on this line. (or other lines, but that's their problem!)